videointelligence/README.rst

```diff
@@ -37,5 +37,5 @@ API enables developers to
 understand the content of a video by encapsulating powerful machine
 learning models in an easy to use REST API.
 
-.. _Video Intelligence: https://cloud.google.com/videointelligence/
-.. _Video Intelligence API docs: https://cloud.google.com/videointelligence/reference/rest/
+.. _Video Intelligence: https://cloud.google.com/video-intelligence/
+.. _Video Intelligence API docs: https://cloud.google.com/video-intelligence/docs/
```

Comment:
@lukesneeringer The issue is not the validity. I was asking if the invalid link

https://cloud.google.com/videointelligence/reference/rest/

should be replaced with

1. https://cloud.google.com/video-intelligence/docs/ `OR`
1. https://cloud.google.com/video-intelligence/docs/reference/rest/